exo-clean/src/cli.ls

```diff
@@ -0,0 +1,47 @@
+require! {
+  'chalk' : {cyan, green, red, blue}
+  'child_process': child-process
+}
+
+module.exports = ->
+
+  if process.argv[2] is "help"
+    return help!
+
+  console.log 'We are about to clean up dangling docker images and volume!\n'
+
+  get-dangling-images-cmd = 'docker images -f dangling=true -q'
+  get-dangling-volumes-cmd = 'docker volume ls -qf dangling=true'
+  remove-images-cmd = 'docker rmi'
+  remove-volumes-cmd = 'docker volume rm'
+  remove-images-msg = "\nremoved all dangling images"
+  remove-volumes-msg = "\nremoved all dangling volumes"
+
+  remove remove-volumes-cmd, get-dangling-volumes-cmd, remove-images-msg
+  remove remove-images-cmd, get-dangling-images-cmd, remove-volumes-msg
```

Comment:
Let's use this package for running Docker processes: https://github.com/apocas/dockerode, and place those commands in `exosphere-shared/src/docker-helper.ls`. We want to avoid running Docker processes through the CLI and use `dockerode` instead. Both the CLI and `dockerode` just makes calls to the Docker HTTP API, but `dockerode` is widely used and well tested and less susceptible to issues we've had in the past with running Docker commands in a separate shell.

I should've been more clear when writing this ticket, so this is my fault. Back when I wrote the ticket we hadn't made the switch to `dockerode` yet and were still running Docker commands through the CLI.